index.js

```diff
@@ -124,6 +132,7 @@ async function runScan({ source, failBuild, acsReportEnable, severityCutoff }) {
   }
 
   const SEVERITY_LIST = ["negligible", "low", "medium", "high", "critical"];
+  const FORMAT_LIST = ["sarif", "json"];
```

Comment:
I think we should probably check that if `outputFormat !== "sarif" && acsReportEnable`, it should be an error.

Reply:
Sounds good. Basically the `acs-report-enable` flag would be an alias for setting the output format to `sarif`, and setting both an error unless they match.

And I actually see no reason not to support the other output formats available on the Grype command line, excepting `template` which gets a lot more complicated (and a potential code injection point depending on what Go templates can do, I'm not familiar with them).

Reply:
Agree -- the other PR I referenced is basically asking for `table` output, so we'll make some changes there for aligning this stuff.